lib/utils/file-searcher.ts

```diff
@@ -23,7 +23,7 @@ export default class FileSearcher {
   }
 
   _getSearchItem(pattern: string): Promise<string[]> {
-    let patterns: string[] = this.searchPatterns[pattern].concat(IGNORE_PATTERNS);
+    let patterns = this.searchPatterns[pattern].concat(IGNORE_PATTERNS);
```

Comment:
I also dropped this type declaration because TS will always infer this type correctly, and now that globby's types are available, this would fail on the next line if it changed.